packages/adapter-mikro-orm/package.json

```diff
@@ -32,22 +32,21 @@
     "dist"
   ],
   "peerDependencies": {
-    "@mikro-orm/core": "^5.0.2",
+    "@mikro-orm/core": "^5.4.1",
```

Comment:
I don't think we should bump peer dependencies, because it will error on some package managers. Is this really a necessary change? If so, this will be a major bump IMO.

Reply:
I don't think its strictly necessary, but will run a test.